src/Raven.Server/Documents/Subscriptions/Stats/SubscriptionBatchPerformanceStats.cs

```diff
@@ -0,0 +1,40 @@
+// -----------------------------------------------------------------------
+//  <copyright file="SubscriptionBatchPerformanceStats.cs" company="Hibernating Rhinos LTD">
+//      Copyright (c) Hibernating Rhinos LTD. All rights reserved.
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using Raven.Server.Documents.Subscriptions.Stats;
+
+namespace Raven.Server.Documents.Subscriptions
+{
+    public class SubscriptionBatchPerformanceStats
+    {
+        public long BatchId { get; set; }
+        public long ConnectionId { get; set; }
+        
+        public long DocumentsCount { get; set; }
+        public long DocumentsSize { get; set; }
+        
+        public long NumberOfIncludedDocuments { get; set; }
+        public long SizeOfIncludedDocuments { get; set; }
+        
+        public long NumberOfIncludedCounters { get; set; }
```

Comment:
DocumentsCount vs NumberOfIncludedCounters vs NumberOfIncludedDocuments. Do we want to unify that?

Do we want to have approx size of counters / ts? Or entire batch size? Right now we hide this info.

Reply:
SubscriptionBatchPerformanceStats has DocumentsCount
SubscriptionBatchRunStats has NumberOfDocuments

Reply:
unified to `NumberOfDocuments `& `SizeOfDocuments`